src/components/AttentionBox/AttentionBox.scss

```diff
@@ -28,43 +30,24 @@
     &-primary {
       @include theme-prop(color, primary-text-color);
       @include theme-prop(background-color, primary-selected-color);
-
-      &__title-container {
-        &__icon {
-          @include theme-prop(color, primary-color);
-        }
-      }
     }
 
     &-danger {
       @include theme-prop(background-color, negative-color-selected);
-
-      &__title-container {
-        &__icon {
-          @include theme-prop(color, negative-color);
-        }
-      }
     }
 
     &-success {
       @include theme-prop(background-color, positive-color-selected);
-
-      &__title-container {
-        &__icon {
-          @include theme-prop(color, positive-color);
-        }
-      }
     }
 
     &-dark {
       @include theme-prop(background-color, primary-background-hover-color);
       @include theme-prop(color, primary-text-color);
+    }
 
-      &__title-container {
-        &__icon {
-          @include theme-prop(color, primary-text-color);
-        }
-      }
+    &-warning {
+      @include theme-prop(background-color, warning-color-selected);
```

Comment:
not a blocker but next time there is no need to use  theme-prop

Reply:
we can just use var and it will do the same thing

Reply:
ok, don't know why we are using theme-prop in a first place)